tableauserverclient/models/schedule_item.py

```diff
@@ -0,0 +1,79 @@
+import xml.etree.ElementTree as ET
+from .. import NAMESPACE
+
+
+class ScheduleItem(object):
+    def __init__(self):
+        self._created_at = None
+        self._end_schedule_at = None
+        self._frequency = None
```

Comment:
Frequency is a string (or, I guess, an enum of a few potential values)

"Weekly", "Daily", "Hourly" there might be another one